src/AppInstallerCommonCore/Manifest/ManifestYamlPopulator.cpp

```diff
@@ -1066,8 +2691,8 @@ namespace AppInstaller::Manifest
             for (auto const& entry : m_p_localizationsNode->Sequence())
             {
                 ManifestLocalization localization;
-                m_p_localization = &localization;
-                auto errors = ValidateAndProcessFields(entry, LocalizationFieldInfos);
+                std::any any = &localization;

```

Comment:
Maybe you should take in the `std::any` by value and just direct construct into the parameter rather than keeping the extra line.